extension/src/wallet/message.guard.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Generated type guards for "message.ts".
+ * WARNING: Do not manually change this file.
+ */
+import { GetWalletRequest, SetWalletRequest, CallRequest } from "./message";
+
+export function isGetWalletRequest(obj: unknown): obj is GetWalletRequest {
+    const typedObj = obj as GetWalletRequest
+    return (
+        (typedObj !== null &&
+            typeof typedObj === "object" ||
+            typeof typedObj === "function") &&
+        typeof typedObj["target"] === "string" &&
+        typedObj["type"] === "get_wallet"
```

Comment:
Could the type changed to `get_wallet_request`?

Reply:
I think this field should be read as ‘request type’ rather than ‘object type’.  So noting that it's a request is redundant.  The fact that it's a request, not a response or some other type of event, should always be clear based on the source of the object — we just have to distinguish it from other types of request.

Reply:
(I'm not super happy with this protocol in general — I'll probably try to move it to [Zod](https://zod.dev/) in the future.  Thankfully in #16 it becomes internal anyway.)